docs/testing.md

```diff
@@ -65,9 +66,13 @@ This just needs the path to the token file (with pre-pended slash), such as `"/t
 
 *TODO (10/24/2017): Unify these token configs!*
 
-### Testing with Travis-CI and Coveralls
+### Testing with Github Actions and Codecov
 
-These tests are run (without credentials) automatically on a pull request to the Narrative Github repo. These are currently run through [Travis-CI](https://travis-ci.org/) and the coverage reported with [Coveralls](https://coveralls.io/). There should be nothing you need to do to make this work.
+These tests are run automatically on a pull request to the Narrative Github repo. These are currently run through [Github Actions](https://docs.github.com/en/free-pro-team@latest/actions) and the coverage reported with [Codecov](https://codecov.io/).
+
+Unit tests are automatically run without credentials, skipping various tests that are, really, more like integration tests.
+
+The integration tests that run with webdriver.io do require an authentication token. This is the `NARRATIVE_TEST_TOKEN` Github secret in the Narrative repo. It will become available in the test environment as `KBASE_TEST_TOKEN`, which is the variable that the `wdio.conf.js` file looks for.
```

Comment:
Can you add a quick description on how to run the tests locally?

Reply:
I updated the docs with local test run info. I think that's in the right place and helpful? It's near the top with the other make directives.